docs/en/cloud/marketplace.md

```diff
@@ -33,37 +34,39 @@ title: CSP Marketplace
 - After successful log in, a new ClickHouse Cloud organization will be created. This organization will be connected to your GCP billing account.
 
 
-### Azure is coming soon
+### Azure (coming soon)
+
+## FAQs
 
-## How can I understand that my organization is connected to the marketplace billing?
+### How can I verify if my organization is connected to CSP marketplace billing?
 
 In ClickHouse Cloud console, navigate to **Admin** -> **Billing**. You should see the name of the marketplace and the link in the **Payment details section**
 
-## I am an existing ClickHouse Cloud user. What will happen if I subscribe to the CH Cloud via aws marketplace?
+### I am an existing ClickHouse Cloud user. What will happen if I subscribe to ClickHouse Cloud via AWS marketplace?
 
 A separate organization connected to the marketplace will be created. Your existing services and organizations will remain and they will not be connected to the marketplace billing.
 
 You can switch between organizations in the top right corner of the ClickHouse Cloud console.
 
-## I am an existing ClickHouse Cloud user and I want my existing services to be billed via marketplace.
+### I am an existing ClickHouse Cloud user and I want my existing services to be billed via marketplace.
 
-Please contact [ClickHouse Cloud support](https://clickhouse.cloud/support) in this case.
+Please follow the steps outlined above to subscribe and sign up via the marketplace. A new organization will be created for you. After this, contact [ClickHouse Cloud support](https://clickhouse.cloud/support) and share the organization ID for the old organization (the one which has existing services and workloads), and the new org (that you created via the marketplace). You will also need to share the email for a user that has the admin role on both orgs.  ClickHouse Cloud support team will then be able to connect your existing services to the new org so that your usage starts getting billed via the marketplace.
```

Comment:
This is incorrect.
The information collected here:
* Org with MP subscription and without services (source org)
* Org with services but no MP subscription (target org)
* Admin user who exists on both

Is used to move the MP subscription from the source org to the target org. We don't move the services as implied by:
> ClickHouse Cloud support team will then be able to connect your existing services to the new org

Reply:
thanks @minodudd - that is what I meant and I understand that is the workflow, let me reword it to clarify.